timesketch/lib/sigma_util.py

```diff
@@ -72,7 +73,11 @@ def get_sigma_rules_path():
         ValueError: If SIGMA_RULES_FOLDERS is not found in the config file.
             or the folders are not readabale.
     """
-    rules_path = current_app.config.get('SIGMA_RULES_FOLDERS', [])
+    try:
+        rules_path = current_app.config.get('SIGMA_RULES_FOLDERS', [])
+    except RuntimeError:
+        # TODO: make that more flexible I have no idea what to best return here
+        return ['/']
```

Comment:
can you please fix this before we submit this PR

Reply:
yes, will raise a ```ValueError```instead. This is mostly if used in unittests, the config file can not be read.